tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+from nomad.config import config
+from nomad.config.models.plugins import SchemaPackageEntryPoint
+
+
+@pytest.fixture(scope='session', autouse=True)
```

Comment:
When the archive parser encounters `m_def: nomad_processing_plugin.SomeSection`, it looks into `Package.registry` to find an alias. If there is no alias, it simply uses `importlib` to load the python package. Therefore, it always works with `nomad_processing_plugin.general` and no alias.

For the alias to work, `Package.registry` must know the alias. But `Package.registry` only has the package where `__init_metainfo__` was called. This happens automatically when the plugin is lazy loaded, e.g. during app startup. But it does not happen automatically during the tests. This fixture is doing this now. 

Same for using `nomad parse`. This also does not lazy load all plugins by default. I will open an MR for nomad-FAIR to fix this in nomad-lab.